shell/app/node_main.cc

```diff
@@ -91,6 +98,19 @@ void SetCrashKeyStub(const std::string& key, const std::string& value) {}
 void ClearCrashKeyStub(const std::string& key) {}
 #endif
 
+#if BUILDFLAG(IS_LINUX)
+int crashpad_handler_pid = -1;
+int crashdump_signal_fd = -1;
```

Comment:
can these be requested from crashpad using `crash_reporter::GetHandlerSocket` the same way they are in the main process, instead of storing these as globals?

Reply:
you are right, `crash_reporter::GetHandlerSocket` will return expected values only if the crashpad client is already connected to the handler process which I think should work for this case as well.

Reply:
Addressed in https://github.com/electron/electron/pull/36460/commits/21b0e1ffadfa7590a9deb814c2bf368c612cb336